src/main/java/net/pms/PMS.java

```diff
@@ -788,6 +788,11 @@ public void run() {
 							"-script", database.getDatabasePath()  + File.separator + "logging_report.txt");
 					} catch (SQLException e) {}
 				}
+
+				// Shut down library scanner
+				if (getConfiguration().getUseCache()) {
+					getDatabase().stopScanLibrary();
+				}
```

Comment:
@SubJunk it seems that the database is already closed when the shutdown hook is processed.